internal/pkg/config/testdata/test-config-alt.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2020 Google LLC
```

Comment:
The name of the file should indicate what the difference from the other `test-config` file is.

Reply:
Updated the name of the file indicate what the difference is.